plasma_framework/test/src/utils/Merkle.test.js

```diff
@@ -0,0 +1,49 @@
+const Merkle = artifacts.require('MerkleWrapper');
+
+const { expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { MerkleTree } = require('../../helpers/merkle.js');
+
+contract('Merkle', () => {
+    beforeEach(async () => {
```

Comment:
beforeEach -> before. We are testing pure functions here, so no need to deploy each time.

Reply:
haha, you're really picky at this XDD